workshops/that_was_easy/README.md

```diff
@@ -1,20 +1,20 @@
 ---
 name: That Was Easy
 description: Website with clickable buttons that make sounds
-author: "@jkwok91"
+author: "@jkwok91 + Yev"
```

Comment:
Yev no

Reply:
@yevbar: it should parse GitHub usernames -- change that to `@jkwok91 + @yevbar`